react/src/views/BoxEdit/BoxEditView.tsx

```diff
@@ -150,10 +157,15 @@ function BoxEditView() {
   const productAndSizesData = data?.products;
   const allTags = data?.tags || null;
 
-  const allLocations = data?.box?.location?.base?.locations.map((location) => ({
-    ...location,
-    name: location.name ?? "",
-  }));
+  const allLocations = data?.box?.location?.base?.locations
```

Comment:
Is it correct that this is a list of all locations with defaultBoxState different from Lost/Scrap?

Reply:
@pylipp yes indeed, this all the locations retrieved from the query and in the next line filter out the Scrap and Lost according to the defaultBoxState.

Reply:
Okay, might it make sense to mention this in a short comment